examples/custom_distribution.ipynb

```diff
@@ -0,0 +1,425 @@
+{
```

Comment:
<p>Can we use generator from, for example scipy?</p><p>(Yes we can, but maybe that should be mentioned explicitly.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Good call. I kind of mention it with this sentence
```
It is meant to be analogous to `scipy.stats.<DIST_NAME>.rvs`.
```
but I should be more explicit about how it's not necessary to hardcode a generator if it already exists in `scipy`.